src/include/daos/dtx.h

```diff
@@ -39,10 +39,14 @@ struct daos_tx_id {
 };
 
 static inline void
-daos_generate_dti(struct daos_tx_id *dti)
+daos_generate_dti(struct daos_tx_id *dti, bool zero)
```

Comment:
it also means it's better to rename daos_copy_dti to daos_dti_copy

Reply:
OK